src/lib/__tests__/InstantSearch-test.js

```diff
@@ -1,19 +1,38 @@
 import algoliasearchHelper from 'algoliasearch-helper';
 import { createSearchClient } from '../../../test/mock/createSearchClient';
+import { createWidget } from '../../../test/mock/createWidget';
+import { createMutliSearchResponse } from '../../../test/mock/createAPIResponse';
+import { runAllMicroTasks } from '../../../test/utils/runAllMicroTasks';
 import InstantSearch from '../InstantSearch';
 import version from '../version';
 
+jest.useFakeTimers();
+
 jest.mock('algoliasearch-helper', () => {
   const module = require.requireActual('algoliasearch-helper');
-
-  return jest.fn((...args) => {
+  const mock = jest.fn((...args) => {
     const helper = module(...args);
 
-    const mock = jest.fn();
-    helper.searchOnlyWithDerivedHelpers = mock;
+    const searchOnlyWithDerivedHelpers = helper.searchOnlyWithDerivedHelpers.bind(
+      helper
+    );
+
+    helper.searchOnlyWithDerivedHelpers = jest.fn((...searchArgs) => {
+      return searchOnlyWithDerivedHelpers(...searchArgs);
+    });
 
     return helper;
   });
+
+  Object.entries(module).forEach(([key, value]) => {
```

Comment:
Since the import is mocked we were not able to access the static values on `algoliasearchHelper`.